python/py-gsd/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-gsd
+version             1.6.1
+revision            0
+categories-append   science
+platforms           darwin
+license             BSD
+
+python.versions     27 36 37
+
+maintainers         {gmail.com:giovanni.bussi @GiovanniBussi} openmaintainer
+
+description         GSD (General Simulation Data) is a file format specification and a library to read and write it.
+long_description    ${description} \
+                    The package also contains a python module that reads and writes hoomd schema gsd files with an easy to use syntax.
+
+homepage            https://gsd.readthedocs.io/
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+
+distname            ${python.rootname}-${version}
+
+checksums           rmd160  df7289b155ce51d97f97bcc524cf881a5f768ff7 \
+                    sha256  4124b1433d552037a9cf8fd769be5d3584ce7e875cea4d3ab48f60bd58e12ca2 \
+                    size    198770
+
+if {${name} ne ${subport}} {
+    depends_build-append port:py${python.version}-setuptools
```

Comment:
whitespacing is incorrect here; the right column should start at a multiple of 4 space (so not at 26, but either at 25 where you wrap the depends_build-append line or at 29).